pkg/cmd/pulumi/deployment_settings_config.go

```diff
@@ -147,7 +168,23 @@ func initializeDeploymentSettingsCmd(cmd *cobra.Command, stack string) (*deploym
 	}
 
 	if !be.SupportsDeployments() {
-		return nil, fmt.Errorf("backends of this type %q do not support deployments",
+		unsuportedBackendMsg := fmt.Sprintf("Backends of type %q do not support managed deployments.\n\n"+
+			"Create a Pulumi Cloud account to get started, learn more about pulumi deployments here: "+
+			"https://www.pulumi.com/docs/pulumi-cloud/deployments/",
+			be.Name())
+
+		unsuportedBackendMsg = colors.Highlight(unsuportedBackendMsg,
+			fmt.Sprintf("Backends of type %q do not support managed deployments", be.Name()),
+			colors.Red+colors.Bold)
+		unsuportedBackendMsg = colors.Highlight(unsuportedBackendMsg, "Pulumi Cloud", colors.BrightCyan+colors.Bold)
```

Comment:
For my education only - do we have a notion of color themes? Some users might find bright cyan hard to see (or incompatible with their terminal settings) but all colors are hard-coded in the code.

Reply:
Not presently I don't, but that's a great idea!

Reply:
That I dont know, I am following the same patterns as other places to keep it consistent